screens/CreateGamePage.js

```diff
@@ -1,59 +1,104 @@
 import React from 'react';
-import { StyleSheet, Text, View, Image, FlatList, Button, Dimensions, TouchableOpacity } from 'react-native';
+import { StyleSheet, Text, View, Image, FlatList, TouchableOpacity, Dimensions } from 'react-native';
 import { Font } from 'expo';
 import { StackNavigator } from 'react-navigation';
+import firebase from 'firebase';
 
 export class CreateScreen extends React.Component {
   static navigationOptions = { header: null };
   state = {
     fontLoaded: false,
-   };
+    pressed: {
+      year: [],
+      place: [],
+    },
+  };
   async componentDidMount() {
     await Font.loadAsync({
       'double-bubble-shadow': require('./../assets/fonts/Double_Bubble_shadow.otf'),
+      'bubble-body': require('./../assets/fonts/Bubbleboddy-FatTrial.ttf'),
       'source-sans-pro': require('./../assets/fonts/source-sans-pro.semibold.ttf'),
       'source-sans-pro-bold': require('./../assets/fonts/SourceSansPro-Bold.ttf'),
+      'bubble-body': require('./../assets/fonts/Bubbleboddy-FatTrial.ttf'),
      });
     this.setState({ fontLoaded: true }) ;
-  }
+  };
+  pressedState = (item) => {
```

Comment:
You should break this function up where you have two: `pressedYear` and `pressedPlace`. Right now your combining two separate logical pieces, you should break them up